engine/audio/src/audiocapture.cpp

```diff
@@ -227,12 +223,20 @@ void AudioCapture::processData()
 #endif
 
     // 5 ********* Calculate the average signal power
-    m_signalPower = pwrSum / m_captureSize;
+    foreach(int barsNumber, m_fftMagnitudeMap.keys())
+    {
+        double bandMagnitude = fillBandsData(barsNumber);
+        pwrSum += bandMagnitude;
+        if (maxMagnitude < bandMagnitude)
+            maxMagnitude = bandMagnitude;
+    }
+
+    m_signalPower = 15 * pwrSum;
+
+    //printf("XXX pwrSum=%f signalpower=%d maxMagn=%f\n", pwrSum, m_signalPower, maxMagnitude);
```

Comment:
I wasn't sure how to scale the result and started dumping values to play with it

Reply:
15 looks like a magic number to me (as well as 10, mentioned in the forums)
I believe, but I'm not sure, the level issue here is all about mic vs line input.
It's probably better to expose a parameter (maybe named 'gain'?) up to the UI, so users can compensate depending on their audio card/input.
Makes sense?

Reply:
I disagree, I think the graph needs to be calibrated such that the maximum level that any sound card can output is 0dbFS (which corresponds to +- 32768 in 16 bit samples).  Can you tell me what the value is for the maximum of the graph, then I can work out what the magic number should be.